modules/ROOT/pages/munit-maven-plugin.adoc

```diff
@@ -5,16 +5,17 @@ endif::[]
 :version-info: 2.0 and newer
 :keywords: munit, testing, unit testing
 :page-aliases: munit-maven-support.adoc, to-set-up-munit-maven-plugin.adoc, faq-working-with-parent-pom.adoc, to-inherit-parent-plugin.adoc, to-ignore-parent-plugin.adoc, to-override-parent-plugin.adoc, munit-maven-plugin-configuration.adoc,to-configure-munit-maven-plugin-maven.adoc
+:page-deployment-options: acb-ide, studio-ide
 
 MUnit provides a Maven plugin that enables you to run your MUnit tests as part of your continuous integration environment.
 
-The following configurations to the `pom.xml` or `settings.xml` files of your Maven installation apply only when running Munit through Maven.
+The following configurations to the `pom.xml` or `settings.xml` files of your Maven installation apply only when running MUnit through Maven.
 
 == Set Up MUnit Maven Plugin
 
-This documentation assumes you have the `<munit.version>` property in your pom.xml.
+You must have the `<munit.version>` property in your pom.xml to set up MUnit Maven Plugin.
```

Comment:
```suggestion
You must have the `{munit.version}` property in your `pom.xml` to set up MUnit Maven Plugin.
```

Reply:
Please verify for this section ☝️

Reply:
Let's see if @egrave can help us with this one.
Thanks!